angular-client/src/pages/landing-page/components/landing-page-connection-display/landing-page-connection-display.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component } from '@angular/core';
+import Storage from 'src/services/storage.service';
+
+@Component({
+  selector: 'landing-page-connection-display',
+  templateUrl: './landing-page-connection-display.component.html',
+  styleUrl: './landing-page-connection-display.component.css'
+})
+export default class LandingPageConnectionDisplay {
+  connected: boolean = false;
+  constructor(private storage: Storage) {}
+
+  ngOnInit() {
+    this.storage.getCurrentRunId().subscribe((runId) => {
```

Comment:
too lit, this is how you get the current run id. The method passed to subscribe is a callback that's called whenever the currentRunId is updated, so you can use that to set a variable that stores the current runId